docs/user-guide/tree_structures.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>One figure for this notebook would be good, possibly from data from this figure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I am not sure that adding a figure would really accomplish much in this notebook, it's showing off tree construction, so visualization feels like it would be a little out of place.</p>